airbyte-integrations/connectors/source-blob/source_blob/spec.json

```diff
@@ -0,0 +1,108 @@
+              "aws_access_key_id": {
+                "type": "string",
+                "description": "In order to access private Buckets stored on AWS S3, this connector would need credentials with the proper permissions. If accessing publicly available data, this field is not necessary."
+              },
+              "aws_secret_access_key": {
+                "type": "string",
+                "description": "In order to access private Buckets stored on AWS S3, this connector would need credentials with the proper permissions. If accessing publicly available data, this field is not necessary.",
+                "airbyte_secret": true
+              },
+              "path_prefix": {
+                "type": "string",
+                "description": "By providing a path-like prefix (e.g. \"myFolder/thisTable/\") to the desired blobs, we can optimise finding these in S3. This is recommended if your bucket contains many folders/files."
+              }
+            }
+          }
+        ]
+      },
+      "format": {
+        "type": "object",
+        "description": "File Format of the file to be replicated (Warning: some format may be experimental, please refer to docs).",
+        "oneOf": [
+          {
+            "title": "csv",
+            "description": "see details <a href=\"https://arrow.apache.org/docs/python/generated/pyarrow.csv.read_csv.html#pyarrow-csv-read-csv\" target=\"_blank\">here</a>",
+            "required": ["filetype"],
+            "properties": {
+              "filetype": {
+                "type": "string",
+                "enum": ["csv"],
+                "default": "csv"
+              },
+              "delimiter": {
+                "type": "string",
+                "description": "leave blank to default to , (comma)"
+              },
+              "quote_char": {
+                "type": "string",
+                "description": "leave blank to default to <strong>\"</strong> (double quote)"
+              },
+              "double_quote": {
+                "type": "boolean",
+                "description": "Whether two quotes in a quoted CSV value denote a single quote in the data. Leave blank to default to True",
+                "default": true
+              },
+              "escape_char": {
+                "type": "string",
+                "description": "The character used for escaping special characters. Leave blank to disallow escape character"
+              },
+              "newlines_in_values": {
+                "type": "boolean",
+                "description": "Whether newline characters are allowed in CSV values. Leave blank to default to False",
+                "default": false
+              },
+              "encoding": {
+                "type": "string",
+                "description": "leave blank to default to <strong>UTF-8</strong> (see <a href=\"https://docs.python.org/3/library/codecs.html#standard-encodings\" target=\"_blank\">list of python encodings</a>)."
```

Comment:
Same here. Can explicitly specify the default value.

Reply:
addressed